commit f3911c5ab93e4295938b2013104d2986ea601454
Author: Peter Korsgaard <jacmet@sunsite.dk>
Date:   Sat Sep 27 09:13:45 2008 +0200

    ALSA: hda: appletv support
    
    The AppleTV needs the same handling as the 24" iMac.
    
    Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Jaroslav Kysela <perex@perex.cz>

Adjusted to apply to Debian's 2.6.26 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c linux-source-2.6.26/sound/pci/hda/patch_realtek.c
--- linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c	2009-01-29 16:43:52.000000000 -0700
+++ linux-source-2.6.26/sound/pci/hda/patch_realtek.c	2009-02-08 23:05:08.000000000 -0700
@@ -6450,6 +6450,7 @@ static int patch_alc882(struct hda_codec
 			board_config = ALC885_MACPRO;
 			break;
 		case 0x106b1000: /* iMac 24 */
+		case 0x106b2800: /* AppleTV */
 			board_config = ALC885_IMAC24;
 			break;
 		case 0x106b00a1: /* Macbook */
